### PR TITLE
Robyn Python project structure and data initialization and validation interfaces added

### PR DESCRIPTION
**Pull request by sumane81**
## Project Robyn

Robyn python version 

Fixes # (issue)

## Type of change
- Robyn Python version project structure initialized with "robynpy" package management
- Data initialization and validation interfaces are added
- robyn.py will be entry point APIs. Initialize method is added in here. All future entry point APIs will go in here
- tutorial1 is added with calling initialize to do integration test.

## How Has This Been Tested?
- Built and installed local package and ran tutorial1 locally.

### CODE REVIEW

File: python/src/tutorials/tutorial1.ipynb
Comment (alxlyj) [inline python/src/tutorials/tutorial1.ipynb:1]: could you share the screencaps for the successful runs on this tutorial notebook?

**Review by alxlyj — APPROVED** [review]
This PR adds data init and validation interfaces for the Robyn port. LGTM.

File: python/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+pip3 install robynpy
+
+## DEV VERSION
+# if you are pulling source from github, install dependencies using requirements.txt
+pip3 install -r requirements.txt
+```
+  
+**2. Getting started**
+
+  * Use this [demo.py](https://github.com/facebookexperimental/Robyn/tree/main/python/src/scripts/demo.py) script as step-by-step guide that is
```
Comment (alxlyj) [inline python/README.md:33]: demo.py does not exist at this time. nit: we might want to flag that this would be included in a future PR?

Comment (sumane81) [reply python/README.md:33]: demo.py will be replaced by set of tutorials. Let me update this.

File: python/src/tutorials/tutorial1.ipynb [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+      "\u001b[0;31mTypeError\u001b[0m: ChannelHyperparameters() takes no arguments"
```
Comment (alxlyj) [inline python/src/tutorials/tutorial1.ipynb:131]: nit: it seems like there was a type error here for this run. was the notebook result updated? 

image

Comment (sumane81) [reply python/src/tutorials/tutorial1.ipynb]: Fixed this.

**Comment by shivkanthb** [comment]
The validation interfaces look good. Good to land!

**Review by shivkanthb — APPROVED** [review]

**Pull request merged**